Error Dockerfile build

Error while building the Dockerfile provided in any of the versions: master, v0.6.0, v0.5.0 and v0.4.0

```
[...] 
[frontend-builder 6/7] COPY frontend/ .                                                                        0.1s
ERROR [frontend-builder 7/7] RUN npm run build                                                                 0.7s

 > [frontend-builder 7/7] RUN npm run build:
#11 0.628 
#11 0.628 > gose@1.0.0 build
#11 0.628 > webpack --config webpack.prod.js
#11 0.628 
#11 0.634 sh: 1: webpack: not found

executor failed running [/bin/sh -c npm run build]: exit code: 127
```

The dockerfile is untouched for this error above and it can be fixed by adding before the npm install:
`RUN npm i -g webpack webpack-cli webpack-merge`

But then the error changes to:

``` [frontend-builder 7/8] COPY frontend/ .                                                                        0.2s
 ERROR [frontend-builder 8/8] RUN npm run build                                                                 0.9s
                                                                                                              
 [frontend-builder 8/8] RUN npm run build:                                                                            
#12 0.571                                                                                                               
#12 0.571 > gose@1.0.0 build                                                                                            
#12 0.571 > webpack --config webpack.prod.js                                                                            
#12 0.571                                                                                                               
#12 0.693 [webpack-cli] Failed to load '/app/webpack.prod.js' config
#12 0.778 [webpack-cli] Error: Cannot find module 'webpack-merge'
#12 0.778 Require stack:
#12 0.778 - /app/webpack.prod.js
#12 0.778 - /usr/local/lib/node_modules/webpack-cli/lib/webpack-cli.js
#12 0.778 - /usr/local/lib/node_modules/webpack-cli/lib/bootstrap.js
#12 0.778 - /usr/local/lib/node_modules/webpack-cli/bin/cli.js
#12 0.778 - /usr/local/lib/node_modules/webpack/bin/webpack.js
#12 0.778     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
#12 0.778     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
#12 0.778     at Module.require (node:internal/modules/cjs/loader:999:19)
#12 0.778     at require (node:internal/modules/cjs/helpers:102:18)
#12 0.778     at Object.<anonymous> (/app/webpack.prod.js:1:19)
#12 0.778     at Module._compile (node:internal/modules/cjs/loader:1099:14)
#12 0.778     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
#12 0.778     at Module.load (node:internal/modules/cjs/loader:975:32)
#12 0.778     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
#12 0.778     at Module.require (node:internal/modules/cjs/loader:999:19) {
#12 0.778   code: 'MODULE_NOT_FOUND',
#12 0.778   requireStack: [
#12 0.778     '/app/webpack.prod.js',
#12 0.778     '/usr/local/lib/node_modules/webpack-cli/lib/webpack-cli.js',
#12 0.778     '/usr/local/lib/node_modules/webpack-cli/lib/bootstrap.js',
#12 0.778     '/usr/local/lib/node_modules/webpack-cli/bin/cli.js',
#12 0.778     '/usr/local/lib/node_modules/webpack/bin/webpack.js'
#12 0.778   ]
#12 0.778 }

executor failed running [/bin/sh -c npm run build]: exit code: 2
```

Development builds succeed by removing
`ENV NODE_ENV=production` and removing `--production` from `RUN npm install --production`

--- a/Dockerfile
+++ b/Dockerfile
@@ -12,16 +12,16 @@ RUN go build -o gose ./cmd
 
 FROM node:17 AS frontend-builder
 
-ENV NODE_ENV=production
-
 WORKDIR /app
 
 COPY frontend/package.json .
 COPY frontend/package-lock.json* .
 
-RUN npm install --production
+RUN npm install
 
 COPY frontend/ .
+
+ENV NODE_ENV=production
 
 RUN npm run build
 

